torch/fx/graph.py

```diff
@@ -923,11 +923,13 @@ def emit_node(node : Node):
                     return
                 qualified_name = _get_qualified_name(node.target)
                 global_name = add_global(qualified_name, node.target)
+                # special case for getattr: node.args could be 2-argument or 3-argument
+                # 2-argument: attribute access; 3-argument: fall through to attrib function call with default value
```

Comment:
nit: To be more precise, this branch handles the case where we have two arguments _and_ the identifier name argument is a valid identifier (i.e. not something like `"0"` that comes from `nn.Sequential`.

Reply:
This is good to know.  The fall through behavior seems to affect the readability of the code.